src/Template/Element/language_dropdown_angular.ctp

```diff
@@ -1,5 +1,4 @@
 <div class="language-dropdown-container"
-     title="{{selectedLanguage.name}}">
```

Comment:
You've also removed the closing `>` for the `div` tag which breaks the HTML code.

Reply:
Thanks for pointing that out. My editor actually highlighted the following `<input` in red, but somehow I failed to notice that...